chapters/order.txt

```diff
@@ -20,6 +20,7 @@ Graphics
 	lines
 	generativemesh
 	advanced_graphics
+  textures
```

Comment:
Also, let me know if there are other things that you think should be covered in a chapter about textures, or if you have some link from the forum that contains material that could be adapted for this chapter